lib/database/classes.ts

```diff
@@ -127,7 +127,7 @@ const getAllClasses = async (): Promise<Class[]> => {
       "SELECT e.name, cl.event_information_id, cl.min_level, cl.max_level, cl.rrstring, cl.start_time, cl.end_time, cl.language, u.id as user_id, u.first_name, u.last_name " +
       "FROM (((event_information e INNER JOIN classes cl ON e.id = cl.event_information_id) " +
       "INNER JOIN commitments ON commitments.event_information_id = e.id) " +
-      " INNER JOIN users u ON commitments.user_id = u.id) WHERE role = 'Teacher'",
+      " INNER JOIN users u ON commitments.user_id = u.id)",
```

Comment:
@Anshul-Birla need to change this for the frontend view of the manage classes view

Reply:
Not sure what to do for this

Reply:
its for me dont worry